xml/ay_bigfile.xml

```diff
@@ -4489,6 +4489,36 @@ config:type="boolean"&gt;true&lt;/diag&gt;</screen>
     <remark role="fixme">Add a short description</remark>
    </para>
 
+   <sect2 xml:id="Software.Selections.base_product">
+    <title>Product Selection</title>
+    <para>
+     Starting with SLE-15 all products are distributed using one media. You have to
+     choose which product you wish to install. You can do it explicitly using.
+     <literal>product</literal> option
+     <example>
+       <title>Explicit product selection</title>
+       <para>
+         &lt;software&gt;&lt;products&gt;&lt;product&gt;SLED15&lt;/product&gt;&lt;/products&gt;&lt;/software&gt;
+       </para>
+     </example>
+    </para>
+    <para>
+      In some special cases it can happen that the media contains just one product. In
+      such case you don't need to select a product explicitly as described above.
+      AutoYaST will select the only available product automatically.
+    </para>
+    <para>
+      This paragraph descibes deprecated behavior which is available to provide
```

Comment:
typo: descibes -> describes

Maybe also reword the first few sentences, because the first part of this para seems like it is longer than need be, proposal: "For backward compatibility with profiles created for pre-&slea; 15 products, &ay; implements a heuristic that selects products automatically. This will be used when the profile does not contain an explicit product selection."

Reply:
my TL asked me to put "deprecated" somewhere around here ;-)